### ISSUE TITLE
Datapoints array empty

### ISSUE THREAD

**Issue by ChaiBapchya** [PredixDev/predix-volcano-app#11]
On Postman
GET - `https://volcano-app.run.aws-usw02-pr.ice.predix.io/api/1.0/datapoints?node=/node/c5e39fa0&sensor=GP_CO2`

returns 
Status `200OK`
Response `[]`

**Comment by j12y** [comment]
You can get the valid nodes, sensors, datatypes just simply using a browser as well:

- https://volcano-app.run.aws-usw02-pr.ice.predix.io/api/1.0/nodes
- https://volcano-app.run.aws-usw02-pr.ice.predix.io/api/1.0/sensors
- https://volcano-app.run.aws-usw02-pr.ice.predix.io/api/1.0/datatypes

For datapoints you just need to specify a valid value -- the value you pasted in the description above looks like an invalid node.  It looks like you have `/node/c5e39fa0` but it should be something like `/node/c5e39fa0-867a-11e6-a353-2f6c041e2491`.

If you paste [this url](https://volcano-app.run.aws-usw02-pr.ice.predix.io/api/1.0/datapoints?node=/node/c5e39fa0-867a-11e6-a353-2f6c041e2491&sensor=GP_CO2) in your browser you should get results:
https://volcano-app.run.aws-usw02-pr.ice.predix.io/api/1.0/datapoints?node=/node/c5e39fa0-867a-11e6-a353-2f6c041e2491&sensor=GP_CO2

**Comment by j12y** [comment]
Even with #10 though this would continue to return an empty set but with a *422* status code instead of *200*.

**Comment by ChaiBapchya** [comment]
Just that 
Here - https://volcano-app.run.aws-usw02-pr.ice.predix.io/docs/html/api/index.html#get--api-1.0-datapoints?node=-node-c5e39fa0&sensor=GP_CO2

The api was wrong!

**Comment by j12y** [comment]
Ah, good catch -- documentation error.

**Comment by ChaiBapchya** [comment]
Exactly! Hence pointed out. Should I pull request the change (if only I can access the code for website/webpage)
